react-components/src/metadata/admin/EditMetadataTemplate.js

```diff
@@ -235,25 +212,57 @@ const validate = values => {
 
     if (!values.name) {
         errors.name = getMessage("required");
-        errors._error = true;
+        errors.error = true;
     }
 
     if (values.attributes && values.attributes.length > 0) {
         const attributesArrayErrors = validateAttributes(values.attributes);
         if (attributesArrayErrors.length > 0) {
             errors.attributes = attributesArrayErrors;
-            errors._error = true;
+            errors.error = true;
         }
     }
 
     return errors;
 };
 
-export default withStoreProvider(
-    reduxForm(
+const handleSubmit = ({ name, description, deleted, attributes },
+                      { props, setSubmitting, setStatus }) => {
+    const resolve = (template) => {
+        setSubmitting(false);
+        setStatus({ success: true, template });
+    };
+    const errorCallback = (httpStatusCode, errorMessage) => {
+        setSubmitting(false);
+        setStatus({ success: false, errorMessage });
+    };
+
+    const { id } = props.template;
+
+    props.presenter.onSaveTemplate(
         {
-            form: ids.METADATA_TEMPLATE_FORM,
-            enableReinitialize: true,
-            validate,
-        }
-    )(withStyles(styles)(withI18N(injectIntl(EditMetadataTemplate), intlData))));
+            id,
+            name,
+            description,
+            deleted,
+            attributes,
+        },
+        resolve,
+        errorCallback,
+    );
+};
+
+const mapPropsToValues = props => {
+    const { template } = props;
+
+    return { ...template };
```

Comment:
The `initialValues` prop is not required by a Formik form (though it will populate its own `initialValues` field in its form props). This `mapPropsToValues` function allows us to populate the form's `values` prop with anything given in the initial props passed to our form component.

For another example:
https://github.com/cyverse-de/ui/blob/299c97c9a4478a4b66a89912ec9ad408eca6a62c/react-components/src/metadata/MetadataTemplateView.js#L421-L468